packages/slice-machine/src/legacy/lib/builders/CustomTypeBuilder/SliceZone/index.tsx

```diff
@@ -466,15 +465,23 @@ const SliceZone: React.FC<SliceZoneProps> = ({
           void completeStep("createSlice");
           syncChanges();
 
-          for (const slice of slices) {
+          for (const { model, langSmithUrl } of slices) {
             void telemetry.track({
```

Comment:
❓  I wonder if Benj would like the langSmith url here as well now that we have it?

Reply:
What do you mean? We're sending the lang smith url as part of the analytics event.